README.md

```diff
@@ -98,6 +99,11 @@ Absolute path, inside the image, where the repository is copied. Defaults
 to `/<repository_name>`. It is also the working directory where commands are
 executed.
 
+#### `optimize_image`
+
+Zero-fill unused filesystem blocks during final cleanup, to make any later
+image compression more efficient. Default is no zero-fill.
```

Comment:
Could we please rephrase this: "Default is to not zero-fill."

Reply:
Done.